components/link.js

```diff
@@ -13,6 +23,14 @@ const LinkComponent = ({ children, locale, ...props }) => {
 
   let href = props.href || pathname;
 
+  if ((props.href && i18nPaths[language] && !i18nPaths[language].includes(href)) || href.startsWith("http")) {
+    return (
+      <Link {...props} href={href} passHref>
+        {children}
+      </Link>
+    );
+  }
```

Comment:
Which usecase is handled in this `if` condition? Kindly specify that in comment.

Reply:
It would handle the cases where a webpage is not yet translated or there is an external website's link